CTOR-825 : Refresh hardware-storage-emc-clariion-navisphere (#3490)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-storage-emc-clariion-navisphere.md

id: hardware-storage-emc-clariion-navisphere
title: EMC Clariion
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Pack assets

### Templates

The Monitoring Connector **EMC Clariion** brings a host template:

* **HW-Storage-EMC-Clariion-Navisphere-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="HW-Storage-EMC-Clariion-Navisphere-custom" label="HW-Storage-EMC-Clariion-Navisphere-custom">

| Service Alias   | Service Template                                          | Service Description                 | Discovery  |
|:----------------|:----------------------------------------------------------|:------------------------------------|:----------:|
| Cache           | HW-Storage-EMC-Clariion-Cache-Navisphere-custom           | Check cache state                   |            |

> | Controller      | HW-Storage-EMC-Clariion-Controller-Navisphere-custom      | Check global controller             |            |

Reviewer: ```suggestion
| Controller      | HW-Storage-EMC-Clariion-Controller-Navisphere-custom      | Check the global Navisphere controller             |            |
```

> | Disks           | HW-Storage-EMC-Clariion-Disks-Navisphere-custom           | Check disks status and performances | X          |

Reviewer: ```suggestion
| Disks           | HW-Storage-EMC-Clariion-Disks-Navisphere-custom           | Check disk status and performance | X          |
```

| *disk_name*#state             | N/A   |
| *disk_name*#hard-read-errors  | N/A   |
| *disk_name*#hard-write-errors | N/A   |
| *disk_name*#read-io           | B/s   |
| *disk_name*#write-io          | B/s   |
| *disk_name*#utils             | %     |

> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.

</TabItem>
<TabItem value="Faults" label="Faults">

| Metric name  | Unit  |
|:-------------|:------|
| array.status | N/A   |

</TabItem>
<TabItem value="Hardware-Global" label="Hardware-Global">

| Metric name     | Unit  |
|:----------------|:------|
| fan.status      | N/A   |
| lcc.status      | N/A   |
| psu.status      | N/A   |
| battery.status  | N/A   |
| memory.status   | N/A   |
| cpu.status      | N/A   |
| iomodule.status | N/A   |
| cable.status    | N/A   |

</TabItem>
<TabItem value="Hba-State" label="Hba-State">

| Metric name      | Unit  |
|:-----------------|:------|
| hba.state.status | N/A   |

</TabItem>
<TabItem value="Port-State" label="Port-State">

| Metric name      | Unit  |
|:-----------------|:------|
| hba.state.status | N/A   |

</TabItem>
</Tabs>

## Prerequisites

> The plugin need the installation of Navisphere client (ask to your EMC Support).

Reviewer: ```suggestion
The plugin needs the Navisphere client to be installed (ask your EMC Support).
```

> The plugin uses by default the path `/opt/Navisphere/bin`.

Reviewer: ```suggestion
 By default , the plugin uses the following path: `/opt/Navisphere/bin`.
```
